src/node_main.cc

```diff
@@ -41,6 +41,7 @@ int wmain(int argc, wchar_t *wargv[]) {
 #else
 // UNIX
 int main(int argc, char *argv[]) {
+  setvbuf(stderr , NULL , _IOLBF , 1024);
```

Comment:
I think there shouldn't be a space immediately after each of the first three arguments?